azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -141,10 +142,6 @@ private void loadAllProjectFlows(final Project project) {
     }
   }
 
-  public List<String> getProjectNames() {
```

Comment:
This method was not used anywhere, so I removed it.

Reply:
Thanks!